rook/scripts/finalise-deploy-rook.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eu -o pipefail
+if [[ "$CLUSTER" = "sc" ]]; then
+  helmfile -e service apply &> /dev/null
+elif [[ "$CLUSTER" = "wc" ]]; then
+  helmfile -e workload apply &> /dev/null
+fi
+exit 0
```

Comment:
Redundant exit command (will automatically exit with zero if it gets here anyways)

```suggestion
```